docs/source/miniconda_releases/23.5.0-1/changes.rst

```diff
@@ -0,0 +1,18 @@
+User-facing changes
+^^^^^^^^^^^^^^^^^^^
```

Comment:
```suggestion
-------------------
```

Reply:
This is in line with the Anaconda release notes. See the following:

```
=============
Release notes
=============

Anaconda 2023.03-1 (April 24, 2023)
-----------------------------------

User-facing changes
^^^^^^^^^^^^^^^^^^^

* Conda has been updated to v23.3.1.

```

Why is Miniconda different?

Reply:
Okay, just hold off on my PR, I will make some changes.